.github/workflows/ci.yml

```diff
@@ -56,5 +56,6 @@ jobs:
           java-package: jre
       - name: Install Dependencies
         run: npm install
+      - run: cd node_modules && cd minecraft-data && mv minecraft-data minecraft-data-old && git clone -b pcp1.21.5 https://github.com/extremeheat/minecraft-data --depth 1 && node bin/generate_data.js
```

Comment:
Combining many commands in one line will be harder to maintain in the future, as git diffs will be less clear and automatic merges will more likely be impeded.
I'd suggest to factor this out into a shell script. That would additionally give opportunity to name magic constants in ways that describe intent.
However, this is easily fixed later in a separate PR. Ping me if I shall make a nice shell script for this.